pom.xml

```diff
@@ -49,21 +49,29 @@
 			<url>https://repo.jenkins-ci.org/public/</url>
 		</pluginRepository>
 	</pluginRepositories>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>io.jenkins.tools.bom</groupId>
+				<artifactId>bom-2.332.x</artifactId>
+				<version>1500.ve4d05cd32975</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>

```

Comment:
Following the [archetype](https://github.com/jenkinsci/archetypes), using the plugin Bill of Materials (BOM) for easier dependency management of plugin versions.